P5/Source/Specs/langKnown.xml

```diff
@@ -63,6 +63,18 @@
       <langKnown tag="fr" level="M">French</langKnown>
     </egXML>
   </exemplum>
+  <exemplum xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#github-mix-mix">
+      <person sex="m" role="speaker, collaborator">
```

Comment:
Need to remove the comma. (See [issue](https://github.com/iljackb/Mixtepec_Mixtec/issues/111)).